pyccel/ast/class_defs.py

```diff
@@ -5,11 +5,15 @@
 """
 This module contains all types which define a python class which is automatically recognised by pyccel
 """
+
+from pyccel.ast.builtin_objects.list_functions import (ListAppend)
```

Comment:
```suggestion
from pyccel.ast.builtin_objects.list_functions import ListAppend
```
These parentheses do nothing

Reply:
For now, yes, but importing additional classes will require putting them in parentheses for convenience

Reply:
Then they can be added when they are necessary. Personally I prefer using multiple lines over grouping inside a tuple. I have never looked into which is faster though. As this is left to personal preference inside the code we should leave it to the person who needs more lines to decide. Besides, as written there is nothing stopping the next person from doing:
```python
from pyccel.ast.builtin_objects.list_functions import (ListAppend), ListPop
```